### DISCUSSION TITLE
Unit test and Automation test for Device-SDK-C

### DISCUSSION THREAD

**Discussion by AnilChoudhury-Eaton** [edgexfoundry/community#92] [General]
Hi @iain-anderson 

Is there any unit test automation framework for the device-sdk-c library?

Anil Choudhury

**Comment by iain-anderson** [comment]
Hi Anil, no there is not. If you go all the way back to the Edinburgh release we had started to use CUnit but the functionality under test got moved out of the SDK and there was no further development on testing
